core/src/main/java/hudson/Functions.java

```diff
@@ -1933,18 +1932,21 @@ public static String toEmailSafeString(String projectName) {
      * @deprecated use {@link JNLPLauncher#getInboundAgentUrl}
      */
     @Deprecated
-    public String getServerName() {
+    public String getServerName() throws URISyntaxException, MalformedURLException {
```

Comment:
this is a breaking API change which you can not do as the method is public and can be used outside the scope of this project.

Reply:
Removed `throws URISyntaxException, MalformedURLException `